cypress/integration/dashboard_spec.js

```diff
@@ -7,7 +7,7 @@
 //Test Suite
 describe('Home Page Dashboard Tests', () => {
 
-    let viewports = [Cypress.env('desktop'),Cypress.env('tablet'),Cypress.env('mobile')];
+    let viewports = [Cypress.env('desktop')];
```

Comment:
are these changes deliberate?

Reply:
Yes, So far from what i have seen there are no code changes between Desktop, tablet and mobile. The app just resizes to fit the screen. It was running the same test 3 times. I can however bring them back.